csaf_2.1/prose/edit/src/introduction-04-informative-references.md

```diff
@@ -16,7 +16,7 @@ CSAF-v2.0
 :    _Common Security Advisory Framework Version 2.0_. Edited by Langley Rock, Stefan Hagen, and Thomas Schmidt. 18 November 2022. OASIS Standard. https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html. Latest stage: https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html.
 
 CVE
-:    _Common Vulnerability and Exposures (CVE) – The Standard for Information Security Vulnerability Names_, MITRE, 1999, https://cve.mitre.org/about/.
+:    _Common Vulnerability and Exposures (CVE) – The Standard for Information Security Vulnerability Names_, MITRE, 1999, https://www.cve.org/About/Overview.
```

Comment:
@sthagen The title changed - should we change that as well? Or use a different site?

Reply:
We could even leave it at the old URL as it is redirected and this is anyway a very fuzzy reference.

@tschmidtb51 what about using the https://cve.mitre.org/docs/cve-intro-handout.pdf URL as this leads to the PDF we intent to reference to, no? ... and we could add the "Revised Feb. 2016" hint I find in that PDF. What do you think?